opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -175,7 +175,7 @@ message Span {
     string name = 2;
 
     // attributes is a collection of attribute key/value pairs on the event.
-    repeated opentelemetry.proto.common.v1.AttributeKeyValue attributes = 3;
+    repeated opentelemetry.proto.common.v1.KeyValue attributes = 3;
```

Comment:
It seems a bit confusing to use the repeated field here when there is a List message defined. I understand the reasoning but would be good to have a comment.

Reply:
Added a comment near `KeyValueList` message definition.